.travis.yml

```diff
@@ -17,15 +17,41 @@ matrix:
       os: linux
       dist: trusty
       sudo: false
+      env:
+        - CHROOT=trusty_i386
+        - CHROOT_DIR=/tmp/chroot/${CHROOT}
       before_install:
         - sudo apt-get update
         - sudo apt-get install -y linux-headers-`uname -r`
         - wget http://mirrors.kernel.org/ubuntu/pool/universe/n/nasm/nasm_2.13.02-0.1_amd64.deb
         - sudo apt-get install -y dpkg
         - sudo dpkg -i nasm_2.13.02-0.1_amd64.deb
+        # Cross-compiling to 32-bit Linux
+        - mkdir -p ${CHROOT_DIR}/haxm
+        - cp -r . ${CHROOT_DIR}/haxm
```

Comment:
nit: If the current directory is our git repo, maybe it would be better to tell `wget` to use a different location for the downloaded files? E.g. `wget -P /tmp/haxm-downloads/ <url>` (BTW, `-P` seems to automatically create the directory). Otherwise this line would copy an extra `.deb` file, and we would be building from a "dirty" source tree.

Reply:
Good point! Fixed as of 2403ad3.